client/state/data-layer/wpcom/concierge/schedules/appointments/detail/test/from-api.js

```diff
@@ -0,0 +1,33 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { SchemaError } from 'state/data-layer/wpcom-http/utils';
+import fromApi from '../from-api';
+
+describe( 'fromApi()', () => {
+	test( 'should validate and transform the data successfully.', () => {
+		const validResponse = { begin_timestamp: 1, end_timestamp: 2, schedule_id: 3 };
+		const expectedResult = { beginTimestamp: 1, endTimestamp: 2, scheduleId: 3 };
+
+		expect( fromApi( validResponse ) ).toEqual( expectedResult );
+	} );
+
+	test( 'should keep not transformed fields.', () => {
```

Comment:
> should keep not transformed fields.

Maybe we could be a bit more precise about what we're testing here. Maybe say something like "should convert expected properties to camel case"

Reply:
And I'm not sure how this test is different from the one above

Reply:
You are right, that description is not clear at all, I repharsed it. The difference between the two tests is that the data in this one also contains the `id` field which is not transformed in any way by the `fromApi` method just passed along.